collect_app/src/main/java/org/odk/collect/android/utilities/DateTimeUtils.java

```diff
@@ -134,7 +134,7 @@ public static DatePickerDetails getDatePickerDetails(String appearance) {
         DatePickerDetails.DatePickerType datePickerType = DatePickerDetails.DatePickerType.GREGORIAN;
         DatePickerDetails.DatePickerMode datePickerMode = DatePickerDetails.DatePickerMode.CALENDAR;
         if (appearance != null) {
-            appearance = appearance.toLowerCase(Locale.US);
+            appearance = appearance.toLowerCase(Locale.getDefault());
```

Comment:
This change is not related to the way we display dates. it should be `Locale.US` like before because we use English words for appearances so we shouldn't use any other languages in `toLowerCase`.

Reply:
Fixed in latest commit. Thanks for pointing out! 😄